src/main/kotlin/com/personio/synthetics/builder/AssertionsBuilder.kt

```diff
@@ -66,8 +66,8 @@ class AssertionsBuilder {
     }
 
     /**
-     * Asserts that the response body contains the given value
-     * @param value Value to look for
+     * asserts that the response body contains the given value
+     * @param value value to look for
```

Comment:
Please use the existing format for consistency - the value description should start from a capital letter